test/manual/amp-list-resize.amp.html

```diff
@@ -0,0 +1,163 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP #0</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <style amp-custom>
+    body {
+      font-family: 'Questrial', Arial;
+    }
+    article {
+      display: block;
+      margin: 8px;
+    }
+
+    amp-list {
+      border: 1px solid green;
+    }
+
+    [overflow] {
+      background: gray;
+      color: #fff;
+      cursor: pointer;
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      z-index: 2;
+      padding: 16px;
+    }
+
+    .story-entry {
+      padding: 40px;
+      border-bottom: 1px solid green;
+    }
+
+    .split {
+      height: 360px;
+    }
+
+    .list {
+      height: 120px;
+    }
+
+    .grid {
+      height: 40px;
+    }
+
+  </style>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <script async custom-element="amp-accordion" src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+</head>
+<body>
+
+    <amp-state id="listStyle">
+      <script type="application/json">
+        "list"
+      </script>
+    </amp-state>
+<article>
+  <h1>AMP List Resize Manual Test</h1>
+
+  <h2>AMP List Basic</h2>
+  <p>Test this under slow internet (throttle to slow 3G).</p>
+  <amp-list width="396" height="80" layout=responsive
```

Comment:
another test with `reset-on-refresh` attribute on `amp-list` might be a good idea. (at min do a quick manual test by setting `reset-on-refresh`.

Reply:
Great idea, will do.

Reply:
Okay, the result of testing `reset-on-refresh` is that we also have to remove the `'i-amphtml-layout-size-defined'` class, which sets the placeholder to be layout fill. This way, the placeholder will take up its original size, and the `amp-list` will be sized to the the placeholder.

Reply:
Relevant changes made.